README.md

```diff
@@ -6,12 +6,14 @@ The Kubernetes Template Project is a template for starting new projects in the G
 - an `OWNERS` with the project leads listed as approvers ([docs on `OWNERS` files][owners])
 - a `CONTRIBUTING.md` outlining how to contribute to the project
 - the `code-of-conduct.md` from this repo, which outlines community behavior and the consequences of breaking the code
-- a `LICENSE` which must be Apache 2.0 for code projects, or Creative Commons 4.0 for documentation repositories
+- a `LICENSE` which must be Apache 2.0 for code projects, or [Creative Commons 4.0] for documentation repositories
 - a `SECURITY_CONTACTS` with the contact points for the Product Security Team 
   to reach out to for triaging and handling of incoming issues. They must agree to abide by the
   [Embargo Policy](https://github.com/kubernetes/sig-release/blob/master/security-release-process-documentation/security-release-process.md#embargo-policy) 
   and will be removed and replaced if they violate that agreement.
 
+  The files `code-of-conduct.md` and `LICENSE` should not contain any custom content.
```

Comment:
>the `code-of-conduct.md` from this repo

I thought this was laid out above.  Maybe specify in the list "an modified copy of `code-of-conduct.md` from this repo, and tack the no custom content onto the end of the LICENSE line?

Reply:
updated.